packages/xo-web/src/xo-app/vm/tab-backups.js

```diff
@@ -0,0 +1,86 @@
+import _ from 'intl'
+import addSubscriptions from 'add-subscriptions'
+import constructQueryString from 'construct-query-string'
+import decorate from 'apply-decorators'
+import React from 'react'
+import { adminOnly } from 'utils'
+import { createPredicate } from 'value-matcher'
+import { injectState, provideState } from 'reaclette'
+import { some } from 'lodash'
+import { subscribeJobs } from 'xo'
+
+import LegacyOverview from '../backup/overview-legacy'
+import { JobsTable } from '../backup/overview'
+
+const ACTIONS = [
+  {
+    handler: (selectedJobs, { goTo }) =>
+      goTo({
+        pathname: '/backup/overview',
+        query: { s: `id: |(${selectedJobs.map(_ => _.id)})` },
+      }),
+    label: _('redirectToBackupJobs'),
+    icon: 'redirect',
+    individualLabel: _('redirectToBackupJob'),
+  },
+]
+
+const INDIVIDUAL_ACTIONS = [
+  {
+    handler: (job, { goTo }) =>
+      goTo({
+        pathname: '/home',
+        query: { t: 'VM', s: constructQueryString(job.vms) },
+      }),
+    label: _('redirectToMatchingVms'),
+    icon: 'preview',
+  },
+]
```

Comment:
I think it could be useful to keep the "Edit" and the "See matching VMs" actions here.